x-pack/plugins/ml/public/application/routing/routes/new_job/wizard.tsx

```diff
@@ -221,7 +222,7 @@ const PageWrapper: FC<WizardPageProps> = ({ location, jobType }) => {
         savedSearchService,
         ANOMALY_DETECTOR
       ),
-    existingJobsAndGroups: mlJobService.getJobAndGroupIds,
+    existingJobsAndGroups: () => mlJobServiceFactory(undefined, mlApiServices).getJobAndGroupIds(),
```

Comment:
Should we see all available groups in the data recognizer wizard?

<img width="1152" alt="Screenshot 2024-04-15 at 12 18 09" src="https://github.com/elastic/kibana/assets/7405507/dbb28a80-0016-43ea-9be6-0168b37aa5ed">

Reply:
Yes, that also uses the same group list. Updated in https://github.com/elastic/kibana/pull/180785/commits/83bc2fd0d370e0d588f704344fb4d5a69cea8c9b